src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -184,6 +184,7 @@ class GazeDataFrame:
 
```

Comment:
I think it's finally safe to remove `valid_pixel_position_columns`, `valid_position_columns`, `valid_velocity_columns`, `valid_acceleration_columns`.

They aren't documented anywhere and are only used by `load_gaze_file()` in `dataset_files.py` for files that we have saved after preprocessing.

Reply:
included valid_... in dataset_files.py